src/features/farming/bakery/components/Cakes.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useContext, useState } from "react";
+import Decimal from "decimal.js-light";
+
+import token from "assets/icons/token.gif";
+
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import { Context } from "features/game/GameProvider";
+
+import { useActor } from "@xstate/react";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { ToastContext } from "features/game/toast/ToastQueueProvider";
+import { CAKES, Craftable } from "features/game/types/craftables";
+
+export const Cakes: React.FC = () => {
+  const [selected, setSelected] = useState<Craftable>(
+    CAKES()["Sunflower Cake"]
+  );
+  const { setToast } = useContext(ToastContext);
+  const { gameService } = useContext(Context);
+  const [
+    {
+      context: { state },
+    },
+  ] = useActor(gameService);
+
+  const inventory = state.inventory;
+
+  const sell = (amount = 1) => {
+    gameService.send("item.sell", {
+      item: selected.name,
+      amount,
+    });
+    setToast({
+      content: "SFL +$" + selected.sellPrice?.mul(amount).toString(),
+    });
+  };
+
+  const handleSellOne = () => {
```

Comment:
Is this a preempt on the ability to own/sell multiple cakes in the future? Would it be better to change this when that is possible rather than build it in now?